examples/nlp/question_answering/question_answering_squad.py

```diff
@@ -248,13 +249,10 @@ def create_pipeline(
 
 if __name__ == "__main__":
     args = parse_args()
-    if not os.path.exists(args.data_dir):
-        raise FileNotFoundError("SQUAD datasets not found. Datasets can be " "obtained using scripts/get_squad.py")
-
-    if not args.version_2_with_negative:
-        args.work_dir = f'{args.work_dir}/squad1.1'
-    else:
-        args.work_dir = f'{args.work_dir}/squad2.0'
+    if not os.path.exists(args.dev_file):
+        raise FileNotFoundError("eval data not found. Datasets can be " "obtained using scripts/get_squad.py")
```

Comment:
we have a few scripts folders, could you please specify that this one is examples/nlp/scripts